README.md

```diff
@@ -201,7 +211,7 @@ background-image:
 
 Updated the `name` of the image to one that is include in a /assets/images/ folder. If you add a new folder please make sure to add to the jekyll-assets paths in the _config.yml
 
-```
+```yaml
```

Comment:
Does using the YAML syntax highlighter make this better? I'm not as worried about this as I am about using bash on the commands at the top of the document but I don't think the colours help that much.